circularprogressbar/src/main/java/com/jackandphantom/circularprogressbar/CircleProgressbar.java

```diff
@@ -1,326 +1,323 @@
      */
-    private int DEFAULT_BACKGROUND_PROGRESS_COLOR = Color.GRAY;
-    /* These two variables are used for foreground progess color
-   * if no color is specified by the user then for default we use these variables
-      * */
     private int foregroundProgressColor;
-    private int DEFAULT_FOREGROUND_PROGRESS_COLOR = Color.BLACK;
-
-    /*Progess is used for progress in progressbar*/
+    private static final int DEFAULT_FOREGROUND_PROGRESS_COLOR = ohos.agp.utils.Color.BLACK.getValue();
+    /* Progress is used for progress in progressbar. */
     private float progress = 0;
-
-    /* start angle is used where the progress should start so according to android it will -90 */
-    private int   startAngle = -90;
-
-    /* sweep angle is used for progress angle in progress bar*/
+    /* start angle is used where the progress should start so according to android it will -90. */
+    private int startAngle = -90;
+    /* sweep angle is used for progress angle in progress bar. */
     private float sweepAngle = 0;
-
-    /* max progress as name implies this is used for finding the max value for progress
-    * default it's value is 100
-    * */
+    /**
+     * max progress as name implies this is used for finding the max value for progress
+     *  default it's value is 100.
+     */
     private float maxProgress = 100;
-
-    /*centerPoint is used for findiding the smallest value between width and height of view */
+    /* centerPoint is used for finding the smallest value between width and height of view. */
     private int centerPoint;
-
-    /* subtractingValues is used to find which one is bigger among the background circle and foreground circle */
+    /* subtractingValues is used to find which one is bigger among the background circle and foreground circle. */
     private int subtractingValue;
-
-    /* As the name says this will used for checking that progress start from clockwise or counter clockwise */
+    /* As the name says this will used for checking that progress start from clockwise or counter clockwise. */
     private boolean clockWise;
-    private int drawRadius, drawOuterRadius;
-
-    /* This is default timing of animation which is used to progresss the progress bar */
-    private int DEFAULT_ANIMATION_DURATION = 1500;
-
-    /*This name says that progress bar should be touchable or not*/
+    private int drawRadius;
+    private int drawOuterRadius;
+    /* This name says that progress bar should be touchable or not. */
     private boolean isTouchEnabled = false;
     private boolean roundedCorner;
+    private OnProgressbarChangeListener onProgressbarChangeListener;
 
-    private  OnProgressbarChangeListener onProgressbarChangeListener;
-
+    /**
+     * This is CircleProgressbar constructor.
+     *
+     * @param context Context
+     */
     public CircleProgressbar(Context context) {
         super(context);
         init();
+        addDrawTask(this);
```

Comment:
place all the setListeners into a private method and call the private method in every constructor to avoid code duplication.

Reply:
Done